include/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics/AtmosphericDrag.hpp

```diff
@@ -0,0 +1,108 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_Flight_System_Dynamics_AtmosphericDrag__
+#define __OpenSpaceToolkit_Astrodynamics_Flight_System_Dynamics_AtmosphericDrag__
+
+#include <OpenSpaceToolkit/Physics/Environment/Objects/Celestial.hpp>
+#include <OpenSpaceToolkit/Physics/Time/Instant.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics.hpp>
+#include <OpenSpaceToolkit/Astrodynamics/Flight/System/SatelliteSystem.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace flight
+{
+namespace system
+{
+namespace dynamics
+{
+
+using ostk::physics::env::obj::Celestial;
+using ostk::physics::time::Instant;
+
+using ostk::astro::flight::system::SatelliteSystem;
+using ostk::astro::flight::system::Dynamics;
+
+/// @brief                      Define the acceleration experienced by a point mass due to atmospheric drag
+
+class AtmosphericDrag : public Dynamics
```

Comment:
So I know it will be heavy but while re-adding those, I was thinking about `AtmosphericDragDynamics` vs `AtmosphericDrag` and `CentralBodyGravityDynamics` etc to show the type of object right away (and know it's really a Dynamics and not a Force or else) and avoid confusing for end users. But it would indeed make the interface simpler to use this current naming. Thoughts? @vishwa2710 @robinpdm @lucas-bremond

Reply:
Yeah fair point, I removed the `*Dynamics` because for other inherited objects we don't really use the parent name.
Like for Orbits, we don't have `KeplerOrbit`, `PropagatedOrbit` etc.

Same with Earth in the various models etc.

IMO if there are conflicts in naming we can always name with the `using A = *` in C++ and `import B as A` in python.

But I'm fine either way as long as we're consistent across the repo.